docs: 730 docs add an index to the guide overview (#731)
argilla-io/distilabel: docs/sections/how_to_guides/index.md

# How-to guides

Welcome to the how-to guides section! Here you will find a collection of guides that will help you get started with Distilabel. We have divided the guides into two categories: basic and advanced. The basic guides will help you get started with the core concepts of Distilabel, while the advanced guides will help you explore more advanced features.

## Basic

<div class="grid cards" markdown>

-   __Define Steps for your Pipeline__

    ---

>     Steps are the building blocks of your pipeline. They can be used to generate data, evaluate models, or perform any other task.

Reviewer: ```suggestion
    Steps are the building blocks of your pipeline. They can be used to generate data, evaluate models, manipulate data, or any other general task.
```

    [:octicons-arrow-right-24: Define Steps](basic/step/index.md)

-   __Define Tasks that rely on LLMs__

    ---

>     Tasks are a special type of step that rely on Language Models (LLMs) to generate data.

Reviewer: ```suggestion
    Tasks are a specific type of step that rely on Language Models (LLMs) to generate data.
```

    [:octicons-arrow-right-24: Define Tasks](basic/task/index.md)

-   __Define LLMs as local or remote models__

    ---

>     LLMs are the core of your tasks. They can be used to outline behavior local or remote models.
> 

Reviewer: It makes sense to remove APIs in the title, but I think that here's useful.

```suggestion
    LLMs are the core of your tasks. They are used to integrate with local models or remote APIs.

```

    [:octicons-arrow-right-24: Define LLMs](basic/llm/index.md)

-   __Execute Steps and Tasks in a Pipeline__

    ---

    Pipeline is where you put all your steps and tasks together to create a workflow.

    [:octicons-arrow-right-24: Execute Pipeline](basic/pipeline/index.md)

</div>

## Advanced

<div class="grid cards" markdown>
-  __Using the Distiset dataset object__

    ---

    Distiset is a dataset object based on the datasets library that can be used to store and manipulate data.

    [:octicons-arrow-right-24: Distiset](advanced/distiset.md)

-  __Export data to Argilla__

    ---

>     Argilla is a platform that can be used to store and share data.
> 

Reviewer: ```suggestion
    Argilla is a platform that can be used to store, search, and apply feedback to datasets.
```
